extras/ome_n3lo/convert_ome_xspace.py

```diff
@@ -0,0 +1,100 @@
+"""Dump a fast  x-space grid of the N3LO transition matrix elements.
+The output file have the structure: x_grid, nf=3, nf=4, nf=5.
+"""
+
+import numpy as np
+from click import progressbar
+from eko.mellin import Path
+from ekore.harmonics import cache as c
+from ekore.operator_matrix_elements.unpolarized.space_like import as3
+from eko.interpolation import lambertgrid
+from scipy import integrate
+
+from large_n_limit import Agg_asymptotic, Aqq_asymptotic
+
+XGRID = lambertgrid(500, 1e-6)
+"""X-grid."""
+
+LOG = 0
+"""Matching threshold displaced ?"""
+
+MAP_ENTRIES = {
+    "gg": (0, 0),
+    "gq": (0, 1),
+    "qg": (1, 0),
+    "qq": (1, 1),
+    "Hg": (2, 0),
+    "Hq": (2, 1),
+    "gH": (0, 2),
+    "HH": (2, 2),
+    "qq_ns": (0, 0),
+}
+
+
+def compute_ome(nf, n, is_singlet):
+    """Get the correct ome from eko."""
+    cache = c.reset()
+    if is_singlet:
+        return as3.A_singlet(n, cache, nf, L=0)
+    else:
+        return as3.A_ns(n, cache, nf, L=0)
+
+
+def compute_xspace_ome(entry, nf, x_grid=XGRID):
+    """Compute the x-space transition matrix element, returns A^3(x)."""
+    mellin_cut = 5e-2
```

Comment:
I only checked `["qq_ns", "Hg", "Hq"]`, but for them I could push that down. I only tried `5e-3`, but we may make it even smaller. This would help again in the large-x region (because there we need to integrate "more")

Reply:
Okay shall I set this to `5e-3` then?

Reply:
Fixed in 00fad3a9